src/main/java/com/bravo/user/dao/model/Payment.java

```diff
@@ -15,6 +13,7 @@ public class Payment {
 
   @Id
   @Column(name = "id")
+  @GeneratedValue(strategy = GenerationType.IDENTITY) // Auto increments ID upon creation
```

Comment:
1. The `id` is a `string` so how would the auto increment work? 
2. The `id` is being set explicitly in the constructor. Does this `@GeneratedValue` overwrite that?

Reply:
You are right. 
Being that ID is a string, this isn't needed. I think I added it out of habit.